nidirect_landing_pages/modules/nidirect_campaign_importer/nidirect_campaign_importer.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'NIDirect Campaign Importer'
+type: module
+description: 'Import Drupal 7 campaign pages as landing pages.'
+core_version_requirement: ^8.8 || ^9
```

Comment:
Ooo, nice. Do we want to work this into our other custom modules too?

Reply:
Yup, I've started changing them, will throw up a PR soon